### DISCUSSION TITLE
what's the use of runs_count in GridView control?

### DISCUSSION THREAD

**Discussion by yunguangli** [flet-dev/flet#3051] [General]
In the documentation, runs_count stands for "The number of children in the cross axis".  But I find it confusing or I misunderstand it.  If I need to show 3 tiles per row, shall I set  runs_count = 3?  Then If I enlarge the window/page in desktop app or increase the max_extent attribute, the number of tiles will increase more than 3, just like the ResponsiveRow.  So what's exactly the use of runs_count here?

**Comment by ndonkoHenri** [comment]
> what's exactly the use of runs_count

It defines a fixed number of elements/cells per row(in a vertical grid) or column(in a horizontal grid).
So, setting it to 4 in a vertical grid for ex, means: I want 4 items per displayed row.

The docs for it will be improved.
